### ISSUE TITLE
Informational: eleventy-plugin-error-overlay crashes 1.0.0-beta.1

### ISSUE THREAD

**Issue by brycewray** [stevenpetryk/eleventy-plugin-error-overlay#3]
Passing this on FYI regarding the Eleventy 1.0.0-beta.1 version released today...

If an existing Eleventy site (or clone thereof in my case) updated to Eleventy 1.0.0-beta.1 already had the `eleventy-plugin-error-overlay` plugin installed, running Eleventy will result in the following console error message (substituting `my-computer` below for actual computer name):

```bash
[11ty] Uncaught exception: (more in DEBUG output)
[11ty] > Cannot read property '__original' of undefined

`TypeError` was thrown:
[11ty]     TypeError: Cannot read property '__original' of undefined
        at overrideStatic (/Users/my-computer/eleventy_site_beta/node_modules/eleventy-plugin-error-overlay/index.js:84:35)
        at Immediate.<anonymous> (/Users/my-computer/eleventy_site_beta/node_modules/eleventy-plugin-error-overlay/index.js:45:7)
        at processImmediate (internal/timers.js:461:21)
```

Removing the plugin resolves the problem.

Environment where used:
- macOS 11.6
- Eleventy version 1.0.0-beta.1

**Comment by MWDelaney** [comment]
I'm having this problem, too!

**Comment by nTraum** [comment]
Hi, thanks for great plugin!

I've upgraded to Eleventy 1.0.0 and I'm getting the same error:

```
[11ty] Uncaught exception: (more in DEBUG output)
[11ty] > Cannot read properties of undefined (reading '__original')

`TypeError` was thrown:
[11ty]     TypeError: Cannot read properties of undefined (reading '__original')
        at overrideStatic (/home/ntraum/coding/ntraum/studiolineatur/node_modules/eleventy-plugin-error-overlay/index.js:84:35)
        at Immediate.<anonymous> (/home/ntraum/coding/ntraum/studiolineatur/node_modules/eleventy-plugin-error-overlay/index.js:45:7)
        at processImmediate (node:internal/timers:473:21)
```

**Comment by patricknelson** [comment]
FYI, might have a fix for this at #4. 😄 

You can install and test my fixed version via:

```
npm i "https://github.com/patricknelson/eleventy-plugin-error-overlay.git#fix-11ty-v1.0.0"
```

**Comment by stevenpetryk** [comment]
Hey, forgot to update this issue! Fixed in v1.0.0.
